models/lfs.go

```diff
@@ -0,0 +1,98 @@
+package models
+
+import (
+	"errors"
+	"github.com/go-xorm/xorm"
+	"time"
+)
+
+type LFSMetaObject struct {
```

Comment:
I don't think there is anything more to store about the object. The object itself does not have any notion of versioning - it's basically just a sha256 hash stored together with the object size. If the tracked file changes, a new meta object will be generated on push (with the new hash).
But maybe I still got you wrong - please don't hesitate to explain your point to me if this is the case :smile: